python/test/gluon/test_frontend.py

```diff
@@ -217,3 +220,40 @@ def test_mbarrier(fresh_knobs):
 } loc(#loc)
 #loc = loc(unknown)
 """)
+
+
+@gluon.jit
+def tcgen05_mma_kernel(nvmma_layout: ttgl.constexpr, acc_layout: ttgl.constexpr):
+    a = ttgl.allocate_shared_memory(ttgl.float16, [128, 128], nvmma_layout)
+    b = ttgl.allocate_shared_memory(ttgl.float16, [128, 128], nvmma_layout)
+    acc = blackwell.allocate_tensor_memory(ttgl.float16, [128, 128], acc_layout)
+    blackwell.tcgen05_mma(a, b, acc)
+
+
+@pytest.mark.skipif(not is_cuda() or torch.cuda.get_device_capability()[0] != 10,
+                    reason="Requires blackwell tensor core")
+def test_tcgen05_mma(fresh_knobs):
+    knobs.compilation.disable_line_info = True
```

Comment:
Is it expected that you modify `knobs` rather than `fresh_knobs` in these tests? (it happens in all of them)

Reply:
Yeah, it's the same thing.